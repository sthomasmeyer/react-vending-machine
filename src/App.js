import React from 'react';
import VendingMachine from './VendingMachine';

const App = () => {
  return (
    <div className='App'>
      <VendingMachine />
    </div>
  );
};

export default App;
