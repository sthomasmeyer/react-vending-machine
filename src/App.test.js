import React from 'react';
import { render } from '@testing-library/react';
import App from './App';
import { MemoryRouter } from 'react-router-dom';

//Smoke test:
it('renders without crashing', () => {
  render(<App />, { wrapper: MemoryRouter });
});

it('matches snapshot', () => {
  const { asFragment } = render(<App />, { wrapper: MemoryRouter });
  expect(asFragment()).toMatchSnapshot();
});
