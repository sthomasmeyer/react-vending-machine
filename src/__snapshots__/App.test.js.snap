// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="VendingMachine"
    >
      <h1>
        A Beautiful Vending Machine
      </h1>
      <nav
        class="VendingMachine-static-link"
      >
        <a
          href="/snacks"
        >
          snacks
        </a>
      </nav>
      <nav
        class="VendingMachine-static-link"
      >
        <a
          href="/drinks"
        >
          drinks
        </a>
      </nav>
    </div>
  </div>
</DocumentFragment>
`;
